kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -114,3 +114,41 @@ namespace          domain   cluster    min  max  goal  current  ready
 weblogic-domain-1  domain1  cluster-1  0    4    2     2        2
 weblogic-domain-1  domain1  cluster-2  0    4    0     0        0
 ```
+
+### Scripts to perform a rolling restart of a WebLogic domain or cluster
```

Comment:
It'd be nice to complete the suite of life-cycle scripts by including one that changes the introspectVersion.  I think the details about its usage are documented in the lifecycle docs...

Reply:
Very nice! That was quick!